modules/cnv-deploying-cnv.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/installing-container-native-virtualization.adoc
+
+[id="cnv-deploying-cnv_{context}"]
+= Deploying {ProductName}
+
+After subscribing to the *KubeVirt HyperConverged Cluster Operator* catalog,
+create the *KubeVirt HyperConverged Cluster Operator Deployment* custom resource
+to deploy {ProductName}.
+
+.Prerequisites
+
+* You must be subscribed to the *KubeVirt HyperConverged Cluster Operator* catalog
+in the `kubevirt-hyperconverged` namespace.
+
+.Procedure
+
+. Navigate to the *Catalog* -> *Installed Operators* page.
+
+. Under *Provided APIs*, locate the *KubeVirt HyperConverged Cluster Operator Deployment*
+custom resource. Click *Create New*.
+
+. Click *Create* to launch {ProductName}.
+
+. Navigate to the *Workloads* -> *Pods* page and monitor the {ProductName} Pods
+until they are all *Running*.
+
+. Start using {ProductName}!
```

Comment:
I'd remove this step. In the assembly, you can create a Next steps section to list what you want users to do after they install.

You can amend the previous step to say something like, "After all the pods display the \*Running\* state, you can access {ProductName}."